### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by jiGApache**
Done:
- provided new api method description
- provided the result of testing of the new method

### CODE REVIEW

File: benchmarks_ptb_xl.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## Benchmarks
     
-All methods were trained on [CPSC 2018](http://2018.icbeb.org/Challenge.html). For benchmarking [PTB-XL](https://physionet.org/content/ptb-xl/1.0.1/) dataset was used.
+All methods except of `ecg_is_normal` were trained on [CPSC 2018](http://2018.icbeb.org/Challenge.html). For benchmarking [PTB-XL](https://physionet.org/content/ptb-xl/1.0.1/) dataset was used.
```
Comment (tanyapole) [inline benchmarks_ptb_xl.md:3]: except for

File: benchmarks_ptb_xl.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+
+Method `ecg_is_normal` was trained, validated and tested on [PTB-XL](https://physionet.org/content/ptb-xl/1.0.1/) dataset. 1-8 folds were used for the training set, 9 fold for the validation set and 10 fold for the test set of the entire dataset, respectively.
```
Comment (tanyapole) [inline benchmarks_ptb_xl.md:5]: Folds 1-8 were used as the training set, fold 9 as the validation set and fold 10 as the test set

File: benchmarks_ptb_xl.md [hunk 3]
```diff
@@ -34,3 +36,29 @@ F1-score is used to evaluate each.
 | BER  | 33.79%  |
 | MI  | 56.55%  |
 
+### ECG embedding classification
```
Comment (tanyapole) [inline benchmarks_ptb_xl.md:39]: Название секции должно быть по рассматриваемому функционалу (выполняемой задаче), а не по методу решения. Т.е. вот эту строку убрать. А 41ю переделать из #### в ###

File: benchmarks_ptb_xl.md [hunk 4]
```diff
@@ -34,3 +36,29 @@ F1-score is used to evaluate each.
...
+Performed with `ecg_is_normal`.
+
+It is possible to specify a separate class of abnormalities for which sample data will be selected:
+ - STTC (ST/T Change)
+ - MI (Myocardial Infarction)
+ - HYP (Hypertrophy)
+ - CD (Conduction Disturbance)
+ - ALL (All available classes)
+
+The following parameters were used to train the KNN-classifier:
```
Comment (tanyapole) [inline benchmarks_ptb_xl.md:52]: Гиперпараметры модели лучше в methods.md отправить.

File: methods.md [hunk 5]
```diff
@@ -65,3 +65,10 @@ All NN-based methods use custom CNN architectures [1]. NN-based diagnostics is e
 [1] - Krizhevsky, Alex & Sutskever, Ilya & Hinton, Geoffrey. (2012). ImageNet Classification with Deep Convolutional Neural Networks. Neural Information Processing Systems. 25. 10.1145/3065386. 
 
 [2] - R. R. Selvaraju, M. Cogswell, A. Das, R. Vedantam, D. Parikh and D. Batra, "Grad-CAM: Visual Explanations from Deep Networks via Gradient-Based Localization," 2017 IEEE International Conference on Computer Vision (ICCV), 2017, pp. 618-626, doi: 10.1109/ICCV.2017.74.
+
+### ECG embedding classification
+ECG classification is performed between normal and abnormal classes. The embeddings which are extracted with help of deep convolutional network are classified by k-nearest-neighbour classifier.
```
Comment (tanyapole) [inline methods.md:70]: Добавьте пожалуйста описание, как строилась нейронка, с помощью которой извлекались эмбеддинги (была взята какая-то модель предобученная на ImageNet, или это нейронка обученная на целевом датасете. если на целевом - то на какой задаче (multiclass classification заболеваний, или какой-то SSL, или еще что-то).
Здесь уместно указать гиперпараметры KNN (если не сложно, распишите как вы их выбрали. Если сложно - забейте).

File: methods.md [hunk 6]
```diff
@@ -65,3 +65,10 @@ All NN-based methods use custom CNN architectures [1]. NN-based diagnostics is e
...
+
+![ste_histogram](images/embedding-classification-accuracy.png)
+
+The KNN classifier is trained on a few shot data, as the greatest improvement in metric performance is observed on a small set of training data.
```
Comment (tanyapole) [inline methods.md:74]: В каком смысле KNN обучался на few shot data? Из этой фразы не понятно, какие именно данные брались для обучения.

**Review by tanyapole — CHANGES_REQUESTED** [review]

**Review by MedAI-Lab-ITMO — APPROVED** [review]

**Review by tanyapole — APPROVED** [review]
